lib/fed/middlewares/tracing/loggers.go

```diff
@@ -0,0 +1,298 @@
+package tracing
+
+import (
+	"fmt"
+	"gfx.cafe/gfx/pggat/lib/fed"
+	packets "gfx.cafe/gfx/pggat/lib/fed/packets/v3.0"
+	"log/slog"
+	"strings"
+)
+
+var packetTypeNames = map[fed.Type]string{
```

Comment:
we should probably just generate this, could add a method on fed.Type like DisplayName or something

Reply:
the correct thing to do is to implement `strings.Stringer`